pkg/components/pluggable/grpc.go

```diff
@@ -35,6 +35,8 @@ type GRPCClient interface {
 	Ping(ctx context.Context, in *proto.PingRequest, opts ...grpc.CallOption) (*proto.PingResponse, error)
 }
 
+type GRPCConnectionDialer = func() (*grpc.ClientConn, error)
```

Comment:
This looks weird with the `=`. I'm surprised it's not a syntax error?

Reply:
Actually both syntax are valid with different meanings, one is type aliasing (this case) which types at both sides are interchangeable, the other (without =) is indeed a different type and should be explicit converted, when required,

for this case type aliasing is enough because I just want to reuse the type definition

Reply:
TIL, thanks!!